packages/devtools_app/lib/src/network/network_model.dart

```diff
@@ -1,3 +1,4 @@
+import 'package:flutter/material.dart';
```

Comment:
nit: this is an import of foundation.dart not material.
Doesn't matter but otherwise made me double take to why material was being used in a data model class.

Reply:
done